src/components/DebugCollateral.tsx

```diff
@@ -0,0 +1,138 @@
+import { useAtomValue } from 'jotai';
```

Comment:
I just moved CollateralChoice.tsx to this file and added the comment. It's just handy to be able to refer to all the vault data.